httpobsdashboard/conf/deviations.json

```diff
@@ -180,5 +180,42 @@
     "tlsobs": {
       "pass": true
     }
+  },
+  "HTTP Only": {
+    "httpobs": {
+      "tests": {
+        "strict-transport-security": {
+          "pass": null,
+          "score_description": "Allow HTTP without an HTTPS redirect.",
+          "score_modifier": 0
+        },
+        "redirection-missing": {
```

Comment:
The test name is `redirection`. You can see what you would be overwritten by going to:

https://observatory.mozilla.org/analyze.html?host=mozilla.org

And looking at the `Observatory.state.results` object. Basically this is directly overwriting the contents of that dictionary/object.